### PR TITLE
InventoryManager.priceToAcquire as retrieve_price

### PR DESCRIPTION
**Pull request by Rinn**
This will return the price that is used when calling `acquire` or `retrieve_item` for the "cheapest" way to acquire said item.

Some things to be aware of how acquire works:
* `valueOfInventory` is taken into account for items in your inventory that determines how an item in your inventory affects the retrieve price. This is a float value that defaults to 1.8 so you may want to set this to 1.0 so it matches mall price.
* historical price is usually used (7 day max) but it will mall search on occasion if the prices are within a 2x threshold of each other to get a more exact difference.

### CODE REVIEW

File: src/net/sourceforge/kolmafia/textui/RuntimeLibrary.java [hunk 1]
```diff
@@ -4608,6 +4617,28 @@ public static Value retrieve_item(ScriptRuntime controller, final Value item) {
...
+    int arg2Value = (int) arg2.intValue();
+
+    boolean countThenItem =
+        arg1.getType().equals(DataTypes.INT_TYPE) || arg1.getType().equals(DataTypes.FLOAT_TYPE);
+
+    int count = countThenItem ? arg1Value : arg2Value;
+    int item = countThenItem ? arg2Value : arg1Value;
+
+    if (count <= 0) {
+      return RuntimeLibrary.continueValue();
```
Comment (Veracity0) [inline src/net/sourceforge/kolmafia/textui/RuntimeLibrary.java:4631]: This can't be right. This method returns an int, but the continueValue is a boolean.

**Review by Veracity0 — CHANGES_REQUESTED** [review]

File: src/net/sourceforge/kolmafia/textui/RuntimeLibrary.java (hunk 1 above)
Comment (Rinn) [reply src/net/sourceforge/kolmafia/textui/RuntimeLibrary.java:4631]: oh my bad I copied that from retrieve_item, it should return 0 or -1 I'll fix it.

**Review by Veracity0 — CHANGES_REQUESTED** [review]
Please use DataTypes.ZERO_VALUE

**Comment by Rinn** [comment]
> Please use DataTypes.ZERO_VALUE

Ah right, sorry. Done.

**Review by Veracity0 — APPROVED** [review]

**Pull request merged**